VideoLocker/src/main/java/org/edx/mobile/module/db/IDatabase.java

```diff
@@ -100,59 +108,94 @@ public Integer getWebOnlyVideosCountByChapter(String enrollmentId, String chapte
 
     /**
      * Returns true if any video downloading is in progress for chapter
+     *
      * @param enrollmentId - course which has the chapter
      * @param chapter
      * @return - Number of Downloaded videos
      */
-    public Boolean isVideoDownloadingInChapter(String enrollmentId, 
-            String chapter, DataCallback<Boolean> callback);
-    
-    
+    public Boolean isVideoDownloadingInChapter(String enrollmentId,
+                                               String chapter, DataCallback<Boolean> callback);
+
+
     /**
-     * Returns dmId's of all downloading videos for given Chapter of logged in user 
+     * Returns dmId's of all downloading videos for given Chapter of logged in user
+     *
      * @param enrollmentId - course which has the chapter
      * @param chapter
      */
-    public List<Long> getDownloadingVideoDmIdsForChapter(String enrollmentId, String chapter, 
-            DataCallback<List<Long>> callback);
-    
-    
+    public List<Long> getDownloadingVideoDmIdsForChapter(String enrollmentId, String chapter,
+                                                         DataCallback<List<Long>> callback);
+
+
     /**
      * Returns true if any video downloading is in progress for Section
+     *
      * @param enrollmentId - course which has the chapter
      * @param chapter
      * @param section
      */
     public Boolean isVideoDownloadingInSection(String enrollmentId, String chapter,
-            String section, DataCallback<Boolean> callback);
-    
-    
+                                               String section, DataCallback<Boolean> callback);
+
+    /**
+     * Returns dmId's of all downloading videos for given section of logged in user
```

Comment:
This description should be changed to:

> Returns **{@link DownloadManager} IDs** of all downloading videos **in a** given section.

"of logged in user" is not necessary, and can be omitted.

These changed apply in other method descriptions as well.